ping/ping_output.c

```diff
@@ -0,0 +1,235 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2024-2025 Georg Pfuetzenreuter <mail+ip@georg-pfuetzenreuter.net>
+ * Copyright (c) 2014-2024 Iputils project
+ * Copyright (c) 1989-2006 The Regents of the University of California
+ */
+
+#include "ping.h"
+#include "stdarg.h"
```

Comment:
Please system headers should be in `<>` => `#include <stdbool.h>`
nit: Also add `#include <stdarg.h>` for `bool` (which I would prefer `bool` over `_Bool`.